### PR TITLE
Add project  description

### PR DESCRIPTION
**Pull request by shivam-sharma7**

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
@melloware @gastaldi WDYT?

File: pom.xml [hunk 1]
```diff
@@ -11,13 +11,15 @@
   <version>999-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>Quarkus - Quinoa - Parent</name>
+   <description>Quinoa is a Quarkus extension</description>
```
Comment (melloware) [inline pom.xml:14]:
```suggestion
  <description>Quinoa is a Quarkus extension which eases the development, the build and serving of single page apps or web components alongside Quarkus. It is possible to use it with a Quarkus backend in a single project.</description>
```

**Review by melloware — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (shivam-sharma7) [reply pom.xml]: @melloware Let's do it mate :)

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
Weird this didn't merge into master.  It looks like you merged from one of your branches to one of your branches.

**Comment by shivam-sharma7** [comment]
> Weird this didn't merge into master. It looks like you merged from one of your branches to one of your branches.

I only opened a patch but yeah i merged this in my fork version then opened this PR

**Pull request merged**